app/routes/login.tsx

```diff
@@ -15,10 +15,45 @@ export const action = async ({ request }: ActionArgs) => {
 
 export default function Login() {
   return (
-    <Form method="post">
-      <Heading>Login</Heading>
-      <Text>You will be logged in (a session will be created for you)</Text>
-      <Button type="submit">Login</Button>
-    </Form>
+    <Center height="container.md" marginTop="28">
+      <Grid templateColumns="repeat(2, 1fr)">
+        <GridItem width="lg" height="xs">
+          <Box color="brand.500">
+            <Heading size="3xl">My</Heading>
+            <Heading size="3xl" marginLeft="16">
+              Custom
+            </Heading>
+            <Heading size="3xl" marginLeft="48">
+              Domain
+            </Heading>
+          </Box>
+          <Text fontSize="3xl" marginTop="10">
+            Reliable DNS serving from Seneca to students
+          </Text>
+        </GridItem>
+        <GridItem border="solid" borderRadius="2xl" borderColor="brand.500">
+          <VStack height="100%">
+            <Box
+              border="solid"
+              display="flex"
+              alignItems="center"
+              justifyContent="center"
+              width="100px"
+              height="100px"
+              marginTop="16"
+            >
+              Icon
+            </Box>
+            <Box flex={1} display="flex" alignItems="center" justifyContent="center">
```

Comment:
Use `Flex` if you want flex?

Reply:
Yeah I actually forgot that Flex component was a thing for chakra